simulation.py

```diff
@@ -1,19 +1,37 @@
 import numpy as np
+import time
+import progressbar
+
+# Total number of agents is 90
+NUM_AGENTS_SOCIAL_QUICK = 10
+NUM_AGENTS_SOCIAL_AVERAGE = 10
+NUM_AGENTS_SOCIAL_LATE = 10
+NUM_AGENTS_AVERAGE_QUICK = 10
+NUM_AGENTS_AVERAGE_AVERAGE = 10
+NUM_AGENTS_AVERAGE_LATE = 10
+NUM_AGENTS_STRICT_QUICK = 10
+NUM_AGENTS_STRICT_AVERAGE = 10
+NUM_AGENTS_STRICT_LATE = 10
 
-NUM_AGENTS_SOCIAL_QUICK = 1
-NUM_AGENTS_SOCIAL_AVERAGE = 1
-NUM_AGENTS_SOCIAL_LATE = 1
-NUM_AGENTS_AVERAGE_QUICK = 1
-NUM_AGENTS_AVERAGE_AVERAGE = 1
-NUM_AGENTS_AVERAGE_LATE = 1
-NUM_AGENTS_STRICT_QUICK = 1
-NUM_AGENTS_STRICT_AVERAGE = 1
-NUM_AGENTS_STRICT_LATE = 1
 NUM_OPTIONS = 10
+
 THRESHOLD_SOCIAL = 25
 THRESHOLD_AVERGE = 50
 THRESHOLD_STRICT = 75
 
+def partition(n, d, depth=0):
+    if d == depth:
+        return [[]]
+    return [
+        item + [i]
+        for i in range(n+1)
+        for item in partition(n-i, d, depth=depth+1)
+		]
+
+n = 9
+d = 9
+Agents_lst = [[n-sum(p)] + p for p in partition(n, d-1)]
```

Comment:
python naming convention voor constants is CAPS_WITH_UNDER. Dit is een constant toch? Anders moet het lower_with_under zijn. Het is nu geen van beide

Reply:
war?

Reply:
wat**

Reply:
https://namingconvention.org/python/

Reply:
ah snap m

Reply:
Dit gaat om Agents_lst, dit moet AGENTS_LST worden of agents_lst, afhankelijk van of het een constant is.